webapp/src/hooks/useTimeDistance.ts

```diff
@@ -0,0 +1,30 @@
+import { useTranslate } from '@tolgee/react';
+
+export function useTimeDistance() {
+  const { t } = useTranslate();
+
+  return (timeInPast: number | string | Date) => {
+    const differenceInSeconds =
```

Comment:
If it's more then a month I would just print the date in short variant.

Reply:
I put there medium which is `23. 1. 2025` but short is `23.01.25` which is imho less readable and we have space for the medium one.